### PR TITLE
Continue jestifying tests

### PR DESCRIPTION
**Pull request by tmashuang**
Ommited 
```
'**/unit-global/*.test.js',
'app/scripts/metamask-controller.test.js',
'app/scripts/controllers/transactions/index.test.js',
'app/scripts/controllers/network/pending-middleware.test.js',
'app/scripts/controllers/permissions/permissions-middleware.test.js',
```

due to a issues with the ganache server instance, poorly written, and general compatibility with jest. Most likely will have to rethink/rewrite these from the ground up.

Pinned jest-enviornment-jsdom to 25 for Uint8Array Buffer issue in jest <= 26, https://github.com/facebook/jest/issues/7780

### CODE REVIEW

File: app/scripts/controllers/permissions/permissions-log-controller.test.js [hunk 1]
```diff
@@ -185,19 +184,26 @@ describe('permissions log', function () {
...
-      assert.ok(
-        entry1.id === id1 &&
-          entry1.response === null &&
-          entry2.id === id2 &&
-          entry2.response === null &&
-          entry3.id === id3 &&
-          entry3.response === null,
-        'all entries should be in correct order and without responses',
-      );
+      // expect(
```
Comment (darkwing) [inline app/scripts/controllers/permissions/permissions-log-controller.test.js:191]: Can we remove this?

File: app/scripts/controllers/permissions/permissions-log-controller.test.js [hunk 2]
```diff
@@ -454,11 +437,7 @@ describe('permissions log', function () {
 
       permHistory = permLog.getHistory();
 
-      assert.deepEqual(
-        permHistory,
-        EXPECTED_HISTORIES.case1[1],
-        'should have correct history',
-      );
+      expect(permHistory).toStrictEqual(EXPECTED_HISTORIES.case1[1]);
```
Comment (darkwing) [inline app/scripts/controllers/permissions/permissions-log-controller.test.js:440]: Are we going to miss comments like `'should have correct history'`?

**Review by darkwing — DISMISSED** [review]
This all looks really modern and improved -- just a few comments!

**Review by SirJosh1987 — APPROVED** [review]

**Comment by SirJosh1987** [comment]
Do I have to use a computer?

**Comment by SirJosh1987** [comment]
F you too

**Review by dan437 — APPROVED** [review]
Great job switching tests many more tests to Jest, I believe it's a great improvement! I have left just one comment for a nice-to-have improvement, otherwise it looks good to me (assuming conflicts are resolved) 🟢

**Review by SirJosh1987 — APPROVED** [review]

**Pull request closed without merging**